### ISSUE TITLE
Export as image 

### ISSUE THREAD

**Issue by vladdu** [pawamoy/dependenpy#56]
It would be useful to be able to export matrices/treemaps/graphs as images (PNG or SVG). This will also allow for more colors and gradients. Graphs can use Graphviz, but for the others it needs to be implemented.

Maybe the presentations should be separated from the core functionality, and implemented as some kind of plugins. This way, people could add their own ways of displaying the results. But this is probably a different feature.

**Comment by pawamoy** [comment]
Good idea! At some point I wanted to hook dependenpy and [pydeps](https://github.com/thebjorn/pydeps) together.
